### ISSUE TITLE
Provide no-ops implementations for unsupported targets

### ISSUE THREAD

**Issue by buenaflor** [getsentry/sentry-kotlin-multiplatform#288]
### Description

To improve compatibility we should provide no-ops for JS and WASM so people can compile their projects without having to separate source sets

**Comment by Legion2** [comment]
We are currently using sentry-kotlin-multiplatform in our shared common source set and now want to add wasm target, we successfully updated all our kotlin common dependencies to support wasm target, but only sentry-kotlin-multiplatform does not support wasm. We would be fine with having a no op for wasm just to be able to compile the source set.

**Comment by buenaflor** [comment]
yup makes total sense, I'll tackle this soon since I am doing some JS work on our Flutter SDK as well :D

**Comment by Skaldebane** [comment]
Hi there, any updates on this?

**Comment by buenaflor** [comment]
Hey no update so far, it is on the roadmap but have pushed it back a bit due to other priorities. sorry for that

**Comment by buenaflor** [comment]
Update: we are currently working on shipping `js` and `wasmJs` stub/no-op support. I expect this to land next week-ish
